pkg/controllers/deployment/default_deployment.go

```diff
@@ -49,12 +38,12 @@ func defaultDeployment(
 	if deployment.Annotations == nil {
 		deployment.Annotations = map[string]string{}
 	}
-	deployment.Annotations[deploymentVersionHashKey] = rvsHashStr
+	deployment.Annotations["operator.openshift.io/rvs-hash"] = rvsHashStr
```

Comment:
my fat fingers want you to keep the constant

Reply:
constants are evil! don't be java!